### DISCUSSION TITLE
이현준 - 세일즈 - 아디다스 세일즈 데이터 분석

### DISCUSSION THREAD

**Discussion by joonda** [dinnercoding/wanted-pre-onboarding-challenge-data#105] [2023년 11월 프리온보딩 데이터 챌린지]
## 도메인 선정 이유 ##
* 매출 데이터나, 판매 증대를 또는 채널 유입 등 고객이 어떻게 들어오는지 흐름에 대한 인사이트를 얻고 싶어 선정하였습니다.

## 구체적인 문제 정의 ## 
* 미국 내의 특정 주나 시 등에서 어떤 상품이 제일 잘 팔리는지에 대해 알고 싶습니다.
* 새로운 판매 전략이나 매출 증대에 대한 인사이트를 얻고 싶습니다.

## 데이터 업로드 ##
* Adidas Sales Dataset
(https://www.kaggle.com/datasets/heemalichaudhari/adidas-sales-dataset/data)

**Comment by ghost** [comment]
현준님! 아디다스 판매와 관련된 데이터에서 전체적인 흐름에 대해 분석해보고자 하시는 것 같네요. 
데이터를 분석해보시면서 인사이트를 얻게 된다면 어떤 문제에 대해 이 문제를 왜 풀어야 하는 것인지 조금 더 구체적으로 정의해보시면 좋을 것 같습니다 :) 

가령,  
미국 내의 특정 주나 시 등에서 어떤 상품이 제일 잘 팔리는가?
-> 이것에 대한 인사이트를 얻었을 경우 어떤 문제를 해결할 수 있는 것인가?
